schema-shared.graphql

```diff
@@ -32,7 +38,7 @@ type CorpusItemAuthor {
 """
 Information about pagination in a connection.
 """
-type PageInfo {
+type PageInfo @shareable {
```

Comment:
how is this different from `external`? with shareable, two sub graphs can own an entity?